modules/op-installing-pipelines-operator-using-the-cli.adoc

```diff
@@ -3,11 +3,10 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="op-installing-pipelines-operator-using-the-cli_{context}"]
-= Installing the {pipelines-shortname} Operator using the CLI
+= Installing the {pipelines-shortname} Operator by using the CLI
 
-You can install {pipelines-title} Operator from the OperatorHub using the CLI.
+You can install {pipelines-title} Operator from the OperatorHub by using the CLI.
 
-[discrete]
 .Procedure
 
 . Create a Subscription object YAML file to subscribe a namespace to the {pipelines-title} Operator,
```

Comment:
```suggestion
. Create a `Subscription` object YAML file to subscribe a namespace to the {pipelines-title} Operator,
```

Reply:
@shivanisathe25 Found this one more small change. Othervise, LGTM! :slightly_smiling_face: